gradoop-hbase/src/main/java/org/gradoop/common/config/GradoopHBaseConfig.java

```diff
@@ -21,21 +21,24 @@
 import org.gradoop.common.model.api.entities.EPGMGraphHead;
 import org.gradoop.common.model.api.entities.EPGMVertex;
 import org.gradoop.common.model.impl.pojo.Edge;
-import org.gradoop.common.model.impl.pojo.GraphHead;
-import org.gradoop.common.model.impl.pojo.Vertex;
-import org.gradoop.common.storage.api.GraphHeadHandler;
-import org.gradoop.common.util.HBaseConstants;
 import org.gradoop.common.model.impl.pojo.EdgeFactory;
+import org.gradoop.common.model.impl.pojo.GraphHead;
```

Comment:
Sorry about that, but I have no other idea to wrap `StoreConfig` or `EPGMStore` here. Would you please give some advice?

Reply:
Because I have to wrap some `Store` and `StoreConfig` which are defined in hbase module, I have to move those to `gradoop-flink`. So some changes in `gradoop-hbase` are necessary 

![image](https://user-images.githubusercontent.com/7821898/39519486-838970d4-4e39-11e8-9d20-80babaaccb8d.png)

![image](https://user-images.githubusercontent.com/7821898/39519534-af2f3b9c-4e39-11e8-8be0-dd8c80eaf354.png)

Reply:
Sorry, you misunderstood my comment. The wrapping is OK and necessary. I meant changing the positions of the import statements are not necessary in such cases. As a reviewer, you see this changes and look for differences in the package names - then you realize that only the position has changed.So everything is fine.